packages/transpiler/src/transformations/transform-constructor.ts

```diff
@@ -52,7 +52,7 @@ export function transformConstructor(
 
   const initializeFuncText = isFullyImplemented
     ? `
-    function initialize(${constructorParameterList}) external initializer {
+    function initialize(${constructorParameterList.replace('string memory', 'string calldata')}) external initializer {
```

Comment:
This is extremely naive implementation and error prone. I am aware of this and it is on the Transpiler improvement list.